Scripts/generic.ps1

```diff
@@ -0,0 +1,12 @@
+. .\Scripts\common.ps1
+
+#### Main
+
+if (${Env:With}) {
+    Update-WingetPackage -wingetPackage ${Env:PackageName} -With ${Env:With} -WebsiteURL ${Env:WebsiteURL}
+}
+else {
+    Update-WingetPackage -wingetPackage ${Env:PackageName} -WebsiteURL ${Env:WebsiteURL}
+}
```

Comment:
Empty "-With" should evaluate to $null and therefore "komac" should be used as default.  
-> no if clause necesarry

Reply:
cool, thought the same however "" is not a valid case in the validation set. So please fix it with a working solution thank you.